testkit/nodekit/node.go

```diff
@@ -38,7 +38,7 @@ func NewNode(
 ) (*nodebuilder.Node, error) {
 	// This is necessary to ensure that the account addresses are correctly prefixed
 	// as in the celestia application.
-	sdkcfg := sdk.GetConfig()
+	sdkcfg := testkit.GetConfig()
```

Comment:
```suggestion
	sdkcfg := sdk.GetConfig()
```
Oops 😅  - we need to keep sdk here as it is a mention to `sdk "github.com/cosmos/cosmos-sdk/types"` import above 


![image](https://user-images.githubusercontent.com/1822365/196455860-4510dea0-8cf2-4c6b-9c6a-ada4884af6e8.png)